### PR TITLE
Test managed executor from CDI producer used by ServletContainerInitializer

### PR DESCRIPTION
**Pull request by njr-11**
Add a test case where a ServletContainerInitializer (and a ServletContextListener) use a ManagedExecutorService.  They can submit tasks fine, however, there is a deadlock with the webcontainer if attempting to wait for the task to run because running the task involves establishing context on the thread and running it as though it were part of the web app, which is currently requiring the web application to have already initialized.   This part is commented out, because a discussion is needed with the webcontainer team about it.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/resources/WEB-INF/beans.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<beans xmlns="http://java.sun.com/xml/ns/javaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/beans_1_0.xsd">
+   <interceptors>
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/resources/WEB-INF/beans.xml:15]: Are these empty tags needed?  `<interceptors>` and `<decorators>`

File: dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/resources/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2017 IBM Corporation and others.
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/resources/WEB-INF/web.xml:3]: Copyright date should be updated.

File: dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/src/concurrent/cdi2/web/ExecutorProducer.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/**
```
Comment (nmittles) [inline dev/com.ibm.ws.concurrent_fat_cdi/test-applications/concurrentCDI2App/src/concurrent/cdi2/web/ExecutorProducer.java:1]: Missing copyright header.

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Pull request merged**